doc/templates/command/template.cpp

```diff
@@ -6,6 +6,13 @@ namespace dnf5 {
 
 using namespace libdnf5::cli;
 
+void TemplateCommand::set_parent_command() {
+    auto * arg_parser_parent_cmd = get_session().get_argument_parser().get_root_command();
+    auto * arg_parser_this_cmd = get_argument_parser_command();
+    arg_parser_parent_cmd->register_command(arg_parser_this_cmd);
+    arg_parser_parent_cmd->get_group("software_management_commands").register_argument(arg_parser_this_cmd);
```

Comment:
I'd also prefer to be able to wrap this line, come to think of it.

Reply:
Although I suppose I could change it to this:

```suggestion
    auto group = arg_parser_parent_cmd->get_group("software_management_commands");
    group.register_argument(arg_parser_this_cmd);
```

Reply:
I did end up making that change, and I also shortened the rest of the variable names to keep the width down. My thinking being, these are local variables in a 5-line function, keeping track of exactly what they represent isn't really an issue. But, let me know if you feel it's a problem.